py-glaredb/examples/show.py

```diff
@@ -0,0 +1,27 @@
+import glaredb
+import polars as pl
+import pandas as pd
+
+con = glaredb.connect()
+df = pl.DataFrame(
+    {
+        "A": [1, 2, 3, 4, 5],
+        "fruits": ["banana", "banana", "apple", "apple", "banana"],
+        "B": [5, 4, 3, 2, 1],
+        "cars": ["beetle", "audi", "beetle", "beetle", "beetle"],
+    }
+)
+con.sql("select * from df where fruits = 'banana'").show(True, 2)
+
+df = pd.DataFrame(
+    {
+        "A": [1, 2, 3, 4, 5],
+        "fruits": ["banana", "banana", "apple", "apple", "banana"],
+        "B": [5, 4, 3, 2, 1],
+        "cars": ["beetle", "audi", "beetle", "beetle", "beetle"],
+    }
+)
+
+con = glaredb.connect()
+
+con.sql("select * from df where fruits = 'banana'").show(True, 4);
```

Comment:
I think the signature should use kwargs. 

`show(n_rows, show_metadata=False)`

such as 
```python
x.show(10)
x.show(n_rows=10)
x.show(10, show_metadata=True)
x.show(10, show_metadata=False)
x.show(n_rows=10, show_metadata=False)
```

Reply:
On second thought... I think this shouldn't have any args. I think having similar functionality to duckdb's [show](https://duckdb.org/docs/api/python/relational_api) would be a good starting point. 

if you want to limit the n_rows, you can just add `limit x` to your sql query.

Reply:
cool. will remove the args